index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="main">
+    <h1 id="title">Servey-Form</h1>
+    <p id="description">Fill the servey form </p>
+    <div class="main-content">
+        <div class="form-content">
+            <form  id="survey-form"action="">
+                <label for="name" id="name-label">Name</label>
+               <input  id="name" type="text" required placeholder="Enter your name" /> <br>
+
+               <label for="email" id="email-label">Email</label>
+               <input id="email" type="email" required placeholder="Enter your email" /><br>
+
+               <label for="number" id="number-label">Enter your rate from 1 to 10</label>
+               <input id="number"type="number" min="1" max="10" required placeholder="Enter your rate" / required> <br>
```

Comment:
I think it would be better to not use more break <br> tags instead you can manage the space using CSS

Reply:
Nice job making your comments phrased as suggestions👍. But let's try to explain why we are requesting the change so the recipient has a better understanding of your request😉. Don't forget we were taught in the [previous lesson on how to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) that we should always explain why we are requesting a change😉.
<img width="719" alt="image" src="https://github.com/MozamelJawad/Microverse-Self-Review-Trial/assets/64643995/084d0244-7ded-4910-a71d-634e85e4e78c">